mbf_costmap_nav/src/mbf_costmap_nav/costmap_navigation_server.cpp

```diff
@@ -817,6 +820,20 @@ bool CostmapNavigationServer::callServiceClearCostmaps(std_srvs::Empty::Request&
   return true;
 }
 
+bool CostmapNavigationServer::callForceUpdateCostmaps(std_srvs::Empty::Request& request,
+                                                      std_srvs::Empty::Response& response)
+{
+  // force update both costmaps
+  for (auto costmap : { local_costmap_ptr_, global_costmap_ptr_ })
```

Comment:
Suggestion: I guess an improvement would be if one can specify in the request which costmap to update? I particularly think force updating the local costmap isn't necessary in most applications because its update rate is usually fairly high.

Reply:
Yes I thought about that but I think it would make the service too verbose.
In the end I think this service is more similar to the clear_costmaps service rather than the check_point_cost service so I decided to keep it as Empty for ease of use

Reply:
Hmm all the services though in [mbf_msgs/srv](https://github.com/magazino/move_base_flex/tree/master/mbf_msgs/srv) have a `uint8 costmap` field; so arguably `clear_costmaps` is the odd one out...

Reply:
I also think being able to clear just a single costmap would be a reasonable requirement, or rather there seems no reason to enforce having to clear both costmaps?

Reply:
clear_costmaps is the odd one because we just copied from move_base,,,,
all new services added to MBF have allow choosing the costmap, so my :+1: for choosing the costmap

Reply:
Upon further discussion we decided to keep the current implementation of both `clear_costmaps` and `update_costmaps` services being Empty